tutordiscovery/templates/discovery/build/discovery/Dockerfile

```diff
@@ -42,6 +42,11 @@ RUN pip install nodeenv==1.7.0
 RUN nodeenv /openedx/nodeenv --node=16.14.2 --prebuilt
 ENV PATH /openedx/nodeenv/bin:${PATH}
 
+# Install the Open edX Atlas translation tool
+RUN curl https://github.com/openedx/openedx-atlas/releases/{{ DISCOVERY_ATLAS_VERSION }}/atlas \
```

Comment:
- We can hardcode the version to 0.1.0.
- Do we need to add the `-L` option?

Reply:
yes